node/flatpak-node-generator.py

```diff
@@ -1317,6 +1384,31 @@ def process_lockfile(self, lockfile: Path) -> Iterator[Package]:
             yield self.parse_package_section(lockfile, section)
 
 
+class YarnRCFileProvider(RCFileProvider):
+    rcfile_name = '.yarnrc'
```

Comment:
Nit, but could you use `ALL_CAPS` for consts? For consistency with the rest of the file (unless I missed one somewhere)